From 070f8e0660bc15af2d0c55291d6dde92688567de Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pali=20Roh=C3=A1r?= <pali.rohar@gmail.com>
Date: Sun, 13 Jan 2013 16:40:52 +0100
Subject: [PATCH 1/3] Rename library libkleo to libkleo_old

* On ubuntu packages depends on one special libkleo4 version, so install it under another name
---
 kaddressbook/editors/CMakeLists.txt |    2 +-
 kmail/CMakeLists.txt                |    2 +-
 libkleo/CMakeLists.txt              |   10 +++++-----
 messageviewer/CMakeLists.txt        |    2 +-
 4 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/kaddressbook/editors/CMakeLists.txt b/kaddressbook/editors/CMakeLists.txt
index 93e30a7..1196bf5 100644
--- a/kaddressbook/editors/CMakeLists.txt
+++ b/kaddressbook/editors/CMakeLists.txt
@@ -14,7 +14,7 @@ kde4_add_plugin(kaddrbk_cryptosettings ${kaddrbk_cryptosettings_PART_SRCS})
 
 
 
-target_link_libraries(kaddrbk_cryptosettings  ${KDE4_KDECORE_LIBS} ${KDEPIMLIBS_KABC_LIBS} kleo kaddressbookprivate kabinterfaces )
+target_link_libraries(kaddrbk_cryptosettings  ${KDE4_KDECORE_LIBS} ${KDEPIMLIBS_KABC_LIBS} kleo_old kaddressbookprivate kabinterfaces )
 
 install(TARGETS kaddrbk_cryptosettings  DESTINATION ${PLUGIN_INSTALL_DIR})
 
diff --git a/kmail/CMakeLists.txt b/kmail/CMakeLists.txt
index 1b8b848..2dbb08a 100644
--- a/kmail/CMakeLists.txt
+++ b/kmail/CMakeLists.txt
@@ -370,7 +370,7 @@ target_link_libraries(kmailprivate
   ${KDE4_KIMAP_LIBRARY}
   ${KDE4_KPARTS_LIBRARY}
   ${KDE4_KRESOURCES_LIBRARY}
-  kleo
+  kleo_old
   ${QGPGME_LIBRARIES}
   mimelib
   ksieve
diff --git a/libkleo/CMakeLists.txt b/libkleo/CMakeLists.txt
index 8e8ed6e..dfe6c60 100644
--- a/libkleo/CMakeLists.txt
+++ b/libkleo/CMakeLists.txt
@@ -108,13 +108,13 @@ if ( NOT KDEPIM_ONLY_KLEO )
    endif ( NOT KLEO_NO_COMPAT )
 endif ( NOT KDEPIM_ONLY_KLEO)
 
-kde4_add_library(kleo SHARED ${kleo_LIB_SRCS})
+kde4_add_library(kleo_old SHARED ${kleo_LIB_SRCS})
 if(WIN32)
-target_link_libraries(kleo ${kleo_LIB_LIBS}  ${GPGME_VANILLA_LIBRARIES} )
+target_link_libraries(kleo_old ${kleo_LIB_LIBS}  ${GPGME_VANILLA_LIBRARIES} )
 else(WIN32)
-target_link_libraries(kleo ${kleo_LIB_LIBS} )
+target_link_libraries(kleo_old ${kleo_LIB_LIBS} )
 endif(WIN32)
-set_target_properties(kleo PROPERTIES VERSION ${GENERIC_LIB_VERSION} SOVERSION ${GENERIC_LIB_SOVERSION} )
-install(TARGETS kleo ${INSTALL_TARGETS_DEFAULT_ARGS})
+set_target_properties(kleo_old PROPERTIES VERSION ${GENERIC_LIB_VERSION} SOVERSION ${GENERIC_LIB_SOVERSION} )
+install(TARGETS kleo_old ${INSTALL_TARGETS_DEFAULT_ARGS})
 
 install ( FILES libkleopatrarc.desktop DESTINATION ${CONFIG_INSTALL_DIR} RENAME libkleopatrarc )
diff --git a/messageviewer/CMakeLists.txt b/messageviewer/CMakeLists.txt
index 15ed7bf..d76f60f 100644
--- a/messageviewer/CMakeLists.txt
+++ b/messageviewer/CMakeLists.txt
@@ -54,7 +54,7 @@ kde4_add_ui_files(libmessageviewer_SRCS
 kde4_add_library(messageviewer SHARED ${libmessageviewer_SRCS})
 
 
-target_link_libraries(messageviewer messagecore kdepim kpgp kleo
+target_link_libraries(messageviewer messagecore kdepim kpgp kleo_old
  ${KDEPIMLIBS_KMIME_LIBS}
  ${KDEPIMLIBS_KPIMUTILS_LIBS} ${KDEPIMLIBS_KABC_LIBS}
  ${KDE4_KDEUI_LIBS} ${KDE4_KHTML_LIBS} ${KDE4_KTNEF_LIBRARY}
-- 
1.7.10.4

